### ISSUE TITLE
Timed out waiting for bootstrap (Error -20) Stuck on reboot after entered download mode

### ISSUE THREAD

**Issue by Riccardo8k** [checkra1n/BugTracker#709]
**Tell us about your setup:**
1. What iDevice are you using? iPad Air 2 A1566 (2018)
2. On what version of iOS is it? 13.3 (Fresh Install from iTunes 3 days ago)
3. What version of checkra1n are you using? GUI or CLI? Both (Safe and not) none of them are working
4. What is your host system (OS version? Hackintosh? VM? etc.)? macOS Catalina 10.15.2
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? USB-C to lightening (not the bugged usb-c cable that prevent DFU, I can enter DFU with this one)

**What are the steps to reproduce the issue?**
1. Start GUI or command line
2. Enter DFU
3. Wait for competition, eventually time out and get error -20
...

**What do you expect, and what is happening instead?**

Expect to finish the process and see code execution and get a jailbreaked device but nothing happen and my device is like before

**Does the issue also occur if you enable Safe Mode?**

Yes in both mode GUI and CL with -s parameter 

**Any other info, error logs, screenshots, ...?**

Verbose logging screenshot 
<img width="737" alt="B7C634B2-8B75-4215-B7C1-F26A637FB8C1" src="https://user-images.githubusercontent.com/59486705/71745969-07c63f80-2e6c-11ea-9907-08fcc7ff84b9.png">
attached

**Comment by Riccardo8k** [comment]
Please tell me how to fix it I really need JB

**Comment by Andre1985g** [comment]
I have the same problem on iPhone X

**Comment by FDX100** [comment]
same error on iphone 6/6s and iphone 5s

**Comment by ghost** [comment]
yeah same made an issue too. try installing itunes or getting ethernet etc.

**Comment by quilenle** [comment]
Maybe the C to the lightning problem.
Would you like to buy a hub then using USB-A to the lightning?

**Comment by Andre1985g** [comment]
I use usb A to lightning right now and it comes the same error...
